src/main/webapp/election-info.js

```diff
@@ -320,3 +365,76 @@ Handlebars.registerHelper('findType', function(locationType){
     return "Standard polling location.";
   }
 })
+
+/**
+ * Handlebars helper that processes information from the FVAP API
+ * into a more understandable deadline for the user
+ * 
+ * @param {String} rule the rule for the deadline, which includes
+ *                      how information should be submitted and
+ *                      what the due date is for (postmarked, etc.)
+ * @param {String} votingType the type of action the voter is taking
+ * 
+ * @return {String} a statement summarizing the rule for the user
+ */
+Handlebars.registerHelper('processRule', function(rule, votingType){
+
+  let submissionType = "";
+  let dueDateType =  "";
+
+  if(rule.includes(":")) {
+    let splitRule = rule.split(":");
+    submissionType = splitRule[0].toLowerCase();
+    dueDateType = splitRule[1].toLowerCase();
+  } else {
+    dueDateType = rule.toLowerCase();
+    submissionType = "";
+  }
+
+  if(votingType == "Ballot Return") {
+    votingType = "a ballot";
+  }
+  if(votingType ==   "Ballot Request") {
+    votingType = "a ballot request";
+  }
+  if(votingType == "Registration") {
+    votingType = "voter registration"
+  }
+
+  votingType = votingType.toLowerCase();
+  votingType = votingType.charAt(0).toUpperCase() + votingType.slice(1);
+
+  dueDateType = dueDateType.replace("*", "");
+
+  return `${votingType} ${submissionType} must be ${dueDateType}`;
+})
+
+/**
+ * Formats the date returned by the FVAP API
+ * (ex. 2020-10-19T00:00:00 to October 19, 2020)
+ * 
+ * @param {String} date the date value provided by the API
+ * 
+ * @return {String} a formatted version of the date
+ */
+Handlebars.registerHelper('formatDate', function(date){
+  let apiDate = date.substring(0, date.length - 9);
+
+  if (apiDate == undefined) {
+    return 'Invalid date';
+  }
+
+  let dateParts = apiDate.split('-');
+
+  if (dateParts.length !== 3) {
+    return 'Invalid date';
+  }
+
+  let monthNum = parseInt(dateParts[1]);
+
+  if (monthNum > MONTHS.length) {
+    return 'Invalid date';
+  }
+
```

Comment:
Is it possible to use Date.parse() instead?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

Reply:
let me look into this!

Reply:
It looks like Date.parse parses a date string and returns the number of milliseconds between the date string and midnight of January 1, 1970. Let me see if this can be easily converted into the string representation that we want to display to the user.

Reply:
nice thanks!